api/src/controllers/fetcher.ts

```diff
@@ -0,0 +1,100 @@
+import { PrismicDocument, Query } from "@prismicio/types";
+import * as prismic from "@prismicio/client";
+import { isString } from "../helpers";
+import { ArticlePrismicDocument, ContentType } from "../types";
+
+const graphQuery = `{
+    articles {
+      title
+      format {
+        title
+      }
+      promo
+      contributors {
+        ...contributorsFields
+        role {
+          title
+        }
+        contributor {
+          ... on people {
+            name
+          }
+          ... on organisations {
+            name
+          }
+        }
+      }
+    }
+  }`.replace(/\n(\s+)/g, "\n");
+
+type GetServerSidePropsPrismicClient = {
+  type: "GetServerSidePropsPrismicClient";
+  client: prismic.Client;
+};
+
+type GetByTypeParams = Parameters<
+  GetServerSidePropsPrismicClient["client"]["getByType"]
+>[1];
```

Comment:
I guess from the name including `ServerSideProps` these have been copied from wc.org - since we're no longer in Next.js land can we change it?

Reply:
I've done a lot of change [here](https://github.com/wellcomecollection/content-api/pull/4/commits/b41e71990140231d4ac4bb8a5b2b992f4cdf3920). Hope this is what you meant by this and [this](https://github.com/wellcomecollection/content-api/pull/4#discussion_r1140120638)?